Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TabbedCommandBar/TabbedCommandBar.bind

```diff
@@ -0,0 +1,16 @@
+﻿<Page xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      mc:Ignorable="d">
+    <Grid>
+        <Border Margin="0">
+            <controls:TabbedCommandBar x:Name="TabbedCommandBarControl">
+                <controls:TabbedCommandBarItem Header="Home">
```

Comment:
Probably want a bit more to this example?

Reply:
Yeah, I know, but at the moment I'm trying to figure out why there are two ribbon items in the sample even though the XAML only defines one.